controllers/feeds/getFeedChat.js

```diff
@@ -225,6 +224,14 @@ const getFeedChatService = async (req, res) => {
       newFeed.push(...mapping);
     }
 
+    if (newFeed.length === 0) {
+      return res.status(200).send({
+        success: true,
+        data: [],
+        message: 'Success get data'
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for an empty `newFeed` array and early return if true is a good practice. It prevents unnecessary processing when there are no new feeds. However, consider providing a more descriptive message in the response to give the client more context about why the data array is empty.
```diff
-         message: 'Success get data'
+         message: 'No new feeds available'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->